.rubocop.yml

```diff
@@ -26,7 +26,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
```

Comment:
- [ ] Update your readme file and explain to the users how can they test your project